tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1011,6 +1011,7 @@ def test_post_notification_with_document_upload(
         template = create_template(service=service, template_type="email", content=content)
 
         statsd_mock = mocker.patch("app.v2.notifications.post_notifications.statsd_client")
+        mocker.patch("app.notifications.validators.check_service_over_bounce_rate")
```

Comment:
I am not certain that these `patch` calls are necessary. Tests were passing fine without them locally but [failing in CI](https://github.com/cds-snc/notification-api/actions/runs/4755087192/jobs/8449991855). When I initially added these I was getting failures but different failures. Re-ran the tests and they passed 🤷.

Reply:
Hmm not too sure why it was inconsistent like that but it would make sense to me you would need to either mock this now that its being called in the post or else ensure whatever it needs to run is present when it does (i.e. `bounce_rate_client`, etc.)